src/FirstSteps/RPG/Game.cs

```diff
@@ -78,5 +53,39 @@ public static void DisplayAllRaces()
                 Console.WriteLine($"{i + 1}. {races[i]}");
             }
         }
+        public static void GetMenu()
+        {
+            var menu = new SelectionPrompt<string>();
+
+            menu.AddChoice("Hero")
+                .AddChild("Stats");
+
+            menu.AddChoiceGroup("Adventures", _adventuresRegistry.GetCommands());
+            var command = AnsiConsole.Prompt(menu);
+            HandleAdventures(command);         
+          
+        }
+        public static void GameMenu()
+        {
+            AnsiConsole.MarkupLine($"{Emoji.Known.BowAndArrow} [darkgreen]Welcome to the RPG game [/] {Emoji.Known.CrossedSwords}");
+
+            Game.CreateHero();
+
+            AnsiConsole.MarkupLine($"{Emoji.Known.Dagger} [red]Let's play the game[/] {Emoji.Known.Dagger}");
+
+            Console.WriteLine("\nType 'end' if you want to quit.");
+            Console.WriteLine("\nIf You want go back to the previous menu press button\n");
+            Console.WriteLine("________________________________");
+            string userInput;
+            do
+            {
+                Game.GetMenu();
```

Comment:
You need to clean this `do` section. Probably you don't need the `userInput` at all because you already read the command in the `GetMenu()` method in line 64 `var command = AnsiConsole.Prompt(menu);`.

Also, you don't need to use `Game.GetMenu();` you can instead write just `GetMenu();` because it is an internal method of `Game.cs`.